test/functional/test_framework/util.py

```diff
@@ -78,6 +78,10 @@ def assert_greater_than(thing1, thing2):
     if thing1 <= thing2:
         raise AssertionError("%s <= %s" % (str(thing1), str(thing2)))
 
+def assert_less_than(*args):
```

Comment:
why so general, the previous one was a simple operator, why obfuscate it as such?

Reply:
that is fair I initially did it like this because there was one scenario where we used two `<` operators but I now switched to using `assert_greater_than` and now broke that statement up to two statements